ansible/library/minigraph_facts.py

```diff
@@ -590,8 +591,9 @@ def main():
         results_clean = json.loads(json.dumps(results, cls=minigraph_encoder))
         module.exit_json(ansible_facts=results_clean)
     except Exception as e:
+        tb = traceback.format_exc()
```

Comment:
Does `traceback` need to be imported?

Reply:
Fixed. I don't know, but it was working.